.changes/config.json

```diff
@@ -3,7 +3,7 @@
   "pkgManagers": {
     "javascript": {
       "version": true,
-      "getPublishedVersion": "npm view ${ pkgFile.pkg.name } version",
+      "getPublishedVersion": "node ../../.scripts/checkIfPublished.mjs ${ pkgFile.pkg.name } ${ pkg.tag ? pkg.tag : 'latest' }",
```

Comment:
Shouldn't this be `node ../.scripts`?

Reply:
Covector runs the commands with the `cwd` in the `./packages/*` directory for each package. So we need to go up two folder levels to the root.

Reply:
Does this mean this won't work if any of the packages are two folders in?

Reply:
I think you mean it will _only_ work for packages that are two folders in. We happen to follow that in this repository though so it should work for the moment.